lib/property.js

```diff
@@ -14,14 +14,17 @@ var utils = require('./utils');
  * @param {String} property
  * @param {String} value
  * @param {Selector} selector the property originates from
+ * @param {Integer} priority 0 for normal properties, 2 for !important properties.
+ * @param {Array} additional array of integers representing more detailed priorities (sorting)
  * @api public
  */
 
-function Property(prop, value, selector, priority) {
+function Property(prop, value, selector, priority, additionalPriority) {
   this.prop = prop;
   this.value = value;
   this.selector = selector;
   this.priority = priority || 0;
+  this.additionalPriority = additionalPriority;
```

Comment:
`this.priority = priority || 0;` indicates that `priority` is an optional parameter, but if that is optional then `addtionalPriority` must be optional and should be `this.additionalPriority = additionalPriority || [];`. Can you check on that and make sure all is consistent?